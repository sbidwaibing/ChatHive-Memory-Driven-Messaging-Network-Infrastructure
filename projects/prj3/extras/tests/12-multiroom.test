+ @zdu test-12 #test1
This is test message 1
.
+ @zdu poems-12 #lewis-carroll #jabberwocky #first-lines
â€™Twas brillig, and the slithy toves
.
+ @zdu test-12 #xx
This is test message 2
.
? test-12
.
? poems-12 5
.
+ @zdu poems-12 #lewis-carroll #the-hunting-of-the-snark #first-lines
"Just the place for a Snark!" the Bellman cried,
.
+ @zdu test-12 #test3
This is test message 3
.
? poems-12
.
? poems-12 5 #lewis-carroll
.
? test-12 #test1
.
? poems-12 2 #jabberwocky
.
? poems-12 1 #lewis-carroll
.
