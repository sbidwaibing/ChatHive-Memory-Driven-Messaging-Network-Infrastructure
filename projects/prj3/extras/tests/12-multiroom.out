ok
ok
ok
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu test-12 #xx
This is test message 2
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu poems-12 #first-lines #jabberwocky #lewis-carroll
’Twas brillig, and the slithy toves
ok
ok
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu poems-12 #first-lines #lewis-carroll #the-hunting-of-the-snark
"Just the place for a Snark!" the Bellman cried,
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu poems-12 #first-lines #lewis-carroll #the-hunting-of-the-snark
"Just the place for a Snark!" the Bellman cried,
YYYY-MM-DDThh:mm:ss.ttt
@zdu poems-12 #first-lines #jabberwocky #lewis-carroll
’Twas brillig, and the slithy toves
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu test-12 #test1
This is test message 1
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu poems-12 #first-lines #jabberwocky #lewis-carroll
’Twas brillig, and the slithy toves
ok
YYYY-MM-DDThh:mm:ss.ttt
@zdu poems-12 #first-lines #lewis-carroll #the-hunting-of-the-snark
"Just the place for a Snark!" the Bellman cried,
